package-builders/Dockerfile.centos8

```diff
@@ -0,0 +1,58 @@
+FROM centos:8
+
+ARG VERSION=0.1
+
+ENV VERSION=$VERSION
+
+ADD package-builders/fedora-build.sh /build.sh
+
+RUN yum install -y 'dnf-command(config-manager)'
+RUN yum config-manager --set-enabled PowerTools
+RUN yum update -y
+RUN yum install -y epel-release
+RUN yum install -y http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/Judy-devel-1.0.5-18.module_el8.1.0+217+4d875839.x86_64.rpm
+RUN yum install -y https://extras.getpagespeed.com/release-el8-latest.rpm
```

Comment:
I assume this is so that we have the `libuv-devel` package available.

Is there some reason we're using this particular repo instead of, say [OKay](https://okay.network/blog-news/rpm-repositories-for-centos-6-and-7.html) (which is what we tell users to use on CentOS 6)?

Also, all the repo stuff should probably be before any package installs and there should generally be an explicit `yum update` following the addition of all the repos.

Reply:
> I assume this is so that we have the `libuv-devel` package available.

Yes; but not just `libuv-devel` other development packages too.

> Is there some reason we're using this particular repo instead of, say [OKay](https://okay.network/blog-news/rpm-repositories-for-centos-6-and-7.html) (which is what we tell users to use on CentOS 6)?

Because this is the Officially supported repos from CentOS / RHEL according to the both teams/orgs.

> Also, all the repo stuff should probably be before any package installs and there should generally be an explicit `yum update` following the addition of all the repos.

Yes I agree with this; I will amend the order and re-test.

Reply:
> Yes I agree with this; I will amend the order and re-test.

Done